csclient/params/params.go

```diff
@@ -23,6 +23,10 @@ const (
 	// EntityIdHeader specifies the header attribute that will hold the
 	// id of the entity for archive GET responses.
 	EntityIdHeader = "Entity-Id"
+
+	// ResourceRevisionHeader specifies the header attribute that will hold the
+	// revision of the resource retrieved by a resource GET request.
+	ResourceRevisionHeader = "Resource-Revision"
```

Comment:
Headers aren't meant to be used for query fields.  Furthermore, the revision should be part of the URL path (in the download request), not a query field.